protocol/communities_messenger_token_permissions_test.go

```diff
@@ -2015,8 +2165,7 @@ func (s *MessengerCommunitiesTokenPermissionsSuite) TestImportDecryptedArchiveMe
 		if !ok || action.ActionType != communities.EncryptionKeyAdd {
 			return false
 		}
-		_, ok = action.Members[common.PubkeyToHex(&s.owner.identity.PublicKey)]
-		return ok
+		return true
```

Comment:
?

Reply:
should not be needed with this change below 🤞 It's the same issue with missing members on `communities.EncryptionKeyAdd`

https://github.com/status-im/status-go/pull/6762/files/c6eb9ebaaabd1615413183f27ced52c1706a7ebd#r2253508272